src/model/websocket.rs

```diff
@@ -14,8 +15,29 @@ pub enum Subscription {
     MiniTickerAll,
     Ticker(String), // symbol
     TickerAll,
-    OrderBook(String, i64), //symbol, depth
-    Depth(String),          //symbol
+    OrderBook(String, i64, UpdateSpeed), //symbol, depth, update speed
+    Depth(String, UpdateSpeed),          //symbol, update speed
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub enum UpdateSpeed {
+    Default,
+    Slow,
+    Fast,
+}
+
+impl std::fmt::Display for UpdateSpeed {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        write!(
+            f,
+            "{}",
+            match self {
+                Self::Default => "",
+                Self::Slow => "@1000ms",
```

Comment:
This seems arbitrary. Are these two values the only accepted ones by Binance? Could you please link their documentation?

Reply:
Yes:
https://github.com/binance-exchange/binance-official-api-docs/blob/master/web-socket-streams.md
Should I put that link in the code as well?